doozer/doozerlib/cli/fbc.py

```
@@ -996,3 +1000,204 @@ async def fbc_rebase_and_build(
+
+        assert runtime.konflux_db is not None, "konflux_db is not initialized. Doozer bug?"
+        runtime.konflux_db.bind(KonfluxBuildRecord)
+
+        if self.major_minor:
+            try:
+                major_str, minor_str = self.major_minor.split(".")
+                ocp_version = (int(major_str), int(minor_str))
+            except (ValueError, AttributeError):
+                raise ValueError(
+                    f"Invalid major-minor format: {self.major_minor}. Expected format: MAJOR.MINOR (e.g. 4.17)"
+                )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)